website/app/routes/$owner.$repo.custom-styles[.]css.ts

```diff
@@ -0,0 +1,23 @@
+import type { LoaderFunction } from 'remix';
+
+async function getConfig({ owner, repo }: Record<string, string>) {
+  const endpoint = `https://raw.githubusercontent.com/${owner}/${repo}/main/docs.json`;
```

Comment:
Should Ref not be taken into account here?

Reply:
yeah for sure.

Reply:
Also I should integrate it with the ProjectConfig type, and provide a default, and probably move it to the config utils file